assets/scripts/menus/HelpMenu.jsx

```diff
@@ -19,8 +19,10 @@ export default class HelpMenu extends React.PureComponent {
   }
 
   onClickAbout () {
-    const mountNode = document.getElementById('dialogs-react')
-    ReactDOM.render(<AboutDialog />, mountNode)
+    store.dispatch({
+      type: SHOW_DIALOG,
+      name: 'ABOUT'
```

Comment:
Ah, right. I actually don't have an opinion either way about this. We can certainly do it that way.